packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -285,9 +290,57 @@ class VmServiceWrapper implements VmService {
         },
       });
     }
+
+    await _addProcessedUrlsToTraceObject(
+        isolateId, resolvedUrlMap, traceObject);
```

Comment:
nit: add a trailing comma after `traceObject` for better formatting. General rule of thumb is if the method call or signature breaks to a second line, we should use the trailing comma, and we should Always use the trailing comma in flutter build methods (with a couple exceptions like a single line SizedBox widget).